third_party/move/move-vm/runtime/src/loader/function.rs

```diff
@@ -315,6 +316,14 @@ impl LoadedFunction {
         }
     }
 
+    /// Returns the module id or, if it is a script, the pseudo module id for scripts.
+    pub fn module_or_script_id(&self) -> &ModuleId {
```

Comment:
I see that you add this here. Do we need to have `Function::module_id` if we have this? We use `None` variant to set the error location to script.